api/v1beta1/machine_webhook.go

```diff
@@ -126,13 +127,13 @@ func (m *Machine) validate(old *Machine) error {
 	if old != nil && old.Spec.ClusterName != m.Spec.ClusterName {
 		allErrs = append(
 			allErrs,
-			field.Invalid(field.NewPath("spec", "clusterName"), m.Spec.ClusterName, "field is immutable"),
+			field.Forbidden(specPath.Child("clusterName"), "spec.clusterName is immutable"),
```

Comment:
I think we can leave the message the same here as the path will be part of the message elsewhere.
```suggestion
			field.Forbidden(specPath.Child("clusterName"), "field is immutable"),
```

Reply:
@killianmuldoon Sounds good, just squashed and pushed.